pkgs/development/python-modules/awscrt/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, cmake, perl, gcc10, mock }:
+
+buildPythonPackage rec {
+  version = "0.11.13";
+  pname = "awscrt";
+
+  buildInputs = [ gcc10 ];
+
+  nativeBuildInputs = [ cmake perl mock ];
+
+  dontUseCmakeConfigure = true;
+
+  postPatch = ''
+    export PATH="${gcc10}/bin:$PATH"
+  '';
+
+  # Unable to import test module
+  doCheck = false;
```

Comment:
Do we need to copy a file over maybe?

Reply:
Possibly. The `test` dir is a module itself and referenced as such, but for some reason an `ImportError` is thrown when it attempts to import it.

Reply:
In the build environment, `test/__init__.py` goes missing. Not sure what is causing this behaviour.

Reply:
Opened an issue: https://github.com/awslabs/aws-crt-python/issues/281